tests/internal_pkg/metrics_incoming_test.go

```diff
@@ -224,3 +224,22 @@ func TestCeilometerIncoming(t *testing.T) {
 		}
 	})
 }
+
+func TestCeilometerGetLabels(t *testing.T) {
+	// test data
+	cm := incoming.CeilometerMetric{
+		Payload: map[string]interface{}{},
+	}
+
+	t.Run("Missing fields", func(t *testing.T) {
```

Comment:
@paramite right here is where I test if there is a panic when none of the fields exist in the map

Reply:
O_o oh I missed this, sorry for that. Interesting... I would promise second ago that this has to panic.